linkerd/app/core/src/metrics/mod.rs

```diff
@@ -67,6 +68,13 @@ pub struct InboundEndpointLabels {
     pub tls: tls::ConditionalServerTls,
     pub authority: Option<http::uri::Authority>,
     pub target_addr: SocketAddr,
+    pub policy: PolicyLabels,
+}
+
+#[derive(Clone, Debug, Default, Eq, PartialEq, Hash)]
+pub struct PolicyLabels {
+    pub server: policy::Labels,
+    pub authz: policy::Labels,
```

Comment:
nit/tioli: kinda seems like this type and it's `FmtLabels` impl should just be provided by the `policy` crate, since there's an identical version of this in the `transport-metrics` crate? not a big deal though.

Reply:
There's just a single instance of this type -- transport-metrics doesn't define any labels, this is all in `app::core` (and it was moved from `core::transport::labels` to here). We could try to deduce in the for loops in a followup but my hunch is that it will be a net gain in LOC.